docs/bundles/SyliusResourceBundle/routing.rst

```diff
@@ -232,3 +232,63 @@ If you want to change that behavior, use the following configuration:
             alias: app.book
             redirect: update
         type: sylius.resource
+
+API Versioning
+--------------
+
+In order to versionate your API responses, you can configure a version of returned responses.
+If you would like to return only second version of your responses, use the following snippet:
+
+.. code-block:: yaml
+
+    app_book:
+        resource: |
+            alias: app.book
+            serialization_version: 2
+        type: sylius.resource_api
+
+Whats more, you can use a path variable to dynamically change your request. You can achieve this by setting a path prefix when importing file or specify it in a path option.
+
+.. code-block:: yaml
+
+    app_book:
+        resource: |
+            alias: app.book
+            serialization_version: $version
+        type: sylius.resource_api
+
+.. note::
+
+    Remember, that a dynamically resolved `books` prefix is not longer available if you will specify ``path``, and it has to be defined manually.
```

Comment:
Either `Remember, a dynamically` or drop the comma.
Also `if you specify` no need for the `will` part.
Not so sure about oxford's commas. Think there was once a discourse to not use them, but since then a lot of docs were rewritten. Personally :+1: in favour, but I guess it varies with taste.